src/main/java/org/jenkinsci/plugins/ewm/steps/ExwsStep.java

```diff
@@ -45,6 +46,10 @@ public DescriptorImpl getDescriptor() {
 
         private List<Template> templates = Collections.emptyList();
 
+        @DataBoundSetter
```

Comment:
normally setters comes after constructors and are grouped together with their getter